lib/Sema/CSDiagnostics.cpp

```diff
@@ -680,7 +680,7 @@ bool MissingConformanceFailure::diagnoseAsAmbiguousOperatorRef() {
   // about missing conformance just in case.
   auto operatorID = name.getIdentifier();
 
-  auto *applyExpr = cast<ApplyExpr>(findParentExpr(anchor));
+  auto *applyExpr = cast_or_null<ApplyExpr>(findParentExpr(anchor));
```

Comment:
There should always be a parent for that anchor otherwise it means we didn't get the outermost parent map...

Reply:
Everywhere else we've been defensive about this, FWIW

Reply:
I think it this case it make sense to use `cast` because there is something else is wrong if we couldn't get to the apply expression.

Reply:
Okay, I'll revert this change in my next PR. Thanks!

Reply:
No rush!